docusaurus.config.js

```diff
@@ -49,6 +49,11 @@ module.exports = {
         },
       ],
     },
+    colorMode: {
+      defaultMode: 'dark',
+      disableSwitch: false,
+      respectPrefersColorScheme: false,
```

Comment:
maybe we can have this to true

what do you think?

respect user preference?

Reply:
with `respectPrefersColorScheme: true` the default mode is ignored, but i think that is a better user experience. the user uses what he wants. 

should i change?

Reply:
please